server-side-rendering/src/routes/+page.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+	export let data;
+
+	const { user } = data;
+
+	function getInitials(name: string) {
+		const [first, last] = name.split(' ');
+		if (last) return `${first[0]}${last[0]}`;
+		return `${first[0]}`;
```

Comment:
Silly question, why no this?
```suggestion
		if (last) return `${first}${last}`;
		return `${first}`;
```

if you split name by space you should get `first=Matej` `last=Baco`

Reply:
Also first could be empty string, we should `?? 'Anonymous` somewhere.